src/ResourceManager/Consumption/Commands.Consumption.Test/ScenarioTests/MarketplaceTests.ps1

```diff
@@ -0,0 +1,181 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
```

Comment:
Cmdlet can return data if the subscription has usage data in Azure Marketplace. If not, it return empty data.

Reply:
Okay, please add instructions to the top of any test that cannot be rerecorded automatically on any subscription.  This could be instructions on things to do in a random subscription to get the correct setup, or the subscription that this needs to be recorded on (and who to contact to either get access to the subscription or to ask to rerecord).  This will make it easier for us to deal with the possible need to rerecord any of the tests.